.bundlewatch.config.js

```diff
@@ -9,10 +9,6 @@ module.exports = {
     repoCurrentBranch: process.env.PR_BRANCH || process.env.PUSH_BRANCH,
   },
   files: [
-    {
-      path: "frontend/admin/dist/app.*.js",
-      maxSize: "325 kB",
-    },
     {
       path: "apps/web/dist/app.*.js",
       maxSize: "250 kB",
```

Comment:
Bump bundle maxSize [because](https://service.bundlewatch.io/results?d=details%7CrepoOwner%7CGCTC-NTGC%7CrepoName%7Cgc-digital-talent%7CrepoCurrentBranch%7Cdebt_5067_migrate-admin%7CrepoBranchBase%7Cmain%7CcommitSha%7C0ee0429ecee8d03d4b6c43bca37a03c0d95170d4%7Cresults%7Cstatus%7Cfail%7CfullResults%7CfilePath%7Capps/web/dist/app.cb87bde8ab900ba5d165.js%7Csize%7CbaseBranchSize%7CmaxSize%7Cfrontend/admin/dist/app.1ff83945327e804d1e35.js%7Cremoved%7Capps/web/dist/app.687842b14e934d4e8738.js%7Csummary%7CmaxSize%2Bcheck%2Bfailed%5E5ZZH%7C0%7C5HJ4%7C0%7C53AO%7C0%7C0%7C5FMK%7C0%5E%5E$0%7C$1%7C2%7C3%7C4%7C5%7C6%7C7%7C8%7C9%7CA%5D%7CB%7C$C%7CD%7CE%7C@$F%7CG%7CC%7CD%7CH%7CP%7CI%7CQ%7CJ%7CR%5D%7C$F%7CK%7CC%7CL%7CH%7CS%7CI%7CT%7CJ%7CU%5D%7C$F%7CM%7CC%7CL%7CH%7CV%7CI%7CW%7CJ%7CX%5D%5D%7CN%7CO%5D%5D) of merge
```suggestion
      maxSize: "300 kB",
```